.solhint.json

```diff
@@ -3,6 +3,7 @@
   "plugins": ["prettier"],
   "rules": {
     "compiler-version": ["error", "^0.6.12"],
-    "prettier/prettier": "error"
+    "prettier/prettier": "error",
+    "reason-string": ["warn",{"maxLength":33}]
```

Comment:
why 33? Seems like an odd value for max line length. Should we use either 79 or 99 as suggested by the [solidity style guid](https://solidity.readthedocs.io/en/v0.5.10/style-guide.html#maximum-line-length)

Reply:
I chose 33 because 32 was the default and I needed 33. Thought we could increase it as necessary, but also happy to go with 79.

Reply:
I think this should stay 32. This reduces the amount of memory needed to encode the string for the error message to one word (why it was chosen to be 32 in the first place).

Reply:
Oops, I thought this was line length.